integration_tests/run_pyspark_from_build.sh

```diff
@@ -112,6 +112,7 @@ else
           --conf 'spark.executor.extraJavaOptions=-ea -Duser.timezone=UTC' \
           --conf 'spark.sql.session.timeZone=UTC' \
           --conf 'spark.sql.shuffle.partitions=12' \
+          --conf "spark.executor.extraClassPath=${ALL_JARS// /:}" \
```

Comment:
Why do we need this in here? Is this for distributed mode? If so then you probably want to 
```
export PYSP_TEST_spark_executor_extraClassPath="${ALL_JARS// /:}"
```
instead so that the version that runs in parallel with pytest-xdist will pick up this change too.

Reply:
yes, it's for `local-cluster` or `standalone` backends